src/App.js

```diff
@@ -33,7 +33,7 @@ function App() {
             <UserProvider>
                 <BrowserRouter>
                     <Header />
-                    <main>
+                    <main className='main'>
```

Comment:
Why you wrote className main for the main component? I didn't saw to use it anywhere.